core/domain/learner_progress_services.py

```diff
@@ -616,3 +650,114 @@ def get_collection_summary_dicts(collection_summaries):
         })
 
     return summary_dicts
+
+
+def get_activity_progress(user_id):
+    """Returns the progress of the learners - the explorations and collections
+    completed by the user and those in progress.
+
+    Args:
+        user_id: str. The id of the learner.
+
+    Returns:
+        LearnerProgress. The learner progress domain object corresponding to the
+            particular learner.
+        dict. The number of entities that are no longer present.
+        list(str). The titles of the collections to which new explorations have
+            been added.
+
+    """
+    incomplete_exploration_ids = get_all_incomplete_exp_ids(user_id)
```

Comment:
Any way you can bundle these four gets into one get-multi, using the new platform service you've just created?